javascript/javascript1/week4/h4.js

```diff
@@ -0,0 +1,97 @@
+let userName = "";
+const toDoList = [];
+
+function getReply(command) {
+    if (typeof command !=='string' || command === "") {
+        return `Error! Not a valide command`;
+    } if (command === "Hello my name is Violeta" && userName === "") {
+        userName = "Violeta";
+        return `Nice to meet you ${userName}`;
+    } else if (command === "Hello my name is Violeta" && userName === ""){
+        userName="";
+        return `Your name is already added`;
+    } if (command === "What is your name?" && userName != "") {
+        return `Your name is ${userName}`;
+   
+    }  else if ( command === "What is your name" && userName === "") {
+        return 'Your name is not mentioned';
+    }  if (command === "Add fishing to my to do list" ) {
+        toDoList.push("fishing");
+        return "Fishing added to your to do list";
+    }  if (command === "Add singing in the shower to my to do list") {
+        toDoList.push("sing in the shower");
+        return "Singing in the shower added to your to do list";
+    } if (command === "Remove fisihing from my to do list") {
+        const indexOftoDolistRemove = toDoList.indexOf("fishing");
+        toDoList.splice(indexOftoDolistRemove, 1);
+        return "Fishing removed from your to do list";
+    }  if (command === "What is on my to do list") {
+        return `There is ${toDoList.length} task left on your list: ${toDoList} `;
+    } if (command === "What day is today?"){
+        const today = new Date();
+        const months = [
+            "Jan",
+            "Feb",
+            "Mar",
+            "Apr",
+            "May",
+            "Jun",
+            "Jul",
+            "Aug",
+            "Sept",
+            "Oct",
+            "Nov",
+            "Dec",
+        ];
+        return `Today is ${today.getDate()} of ${months[today.getMonth()]} ${today.getFullYear()}`;
+    
+    }  if (command.includes("what is")) {
+
+        let matches = command.match(/\d+/g);
+        let oprator = command.split(" ");
+        let result;
+        if (oprator.includes('*')) {
+            result = parseInt(matches[0]) * parseInt(matches[1]);
+            return result;
+        }
+        else if (oprator.includes('+')) {
+            result = parseInt(matches[0]) + parseInt(matches[1]);
+            return result;
+        }
+        else if (oprator.includes('-')) {
+            result = parseInt(matches[0]) - parseInt(matches[1]);
+            return result;
+        }
+        else if (oprator.includes('/')) {
+            result = parseInt(matches[0]) / parseInt(matches[1]);
+            return result;
+        }
```

Comment:
This should be a function that would be called here and it would return results based on numbers and operator. But nice job 👍